From 25a880ecb40f3374721cacf23e878883897b7237 Mon Sep 17 00:00:00 2001
From: Vilas R K <vilas.r.k@intel.com>
Date: Tue, 1 Aug 2023 13:48:52 +0530
Subject: [PATCH] SELinux change for AIDL Conversion of Thermal HAL

This change will add file_context for AIDL
Thermal HAL.

Tracked-On: OAM-110601
Signed-off-by: Vilas R K <vilas.r.k@intel.com>
---
 thermal/thermal-daemon/file_contexts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/thermal/thermal-daemon/file_contexts b/thermal/thermal-daemon/file_contexts
index e4cd421..9c81aee 100644
--- a/thermal/thermal-daemon/file_contexts
+++ b/thermal/thermal-daemon/file_contexts
@@ -4,3 +4,4 @@
 
 # thermal service
 /vendor/bin/hw/android\.hardware\.thermal@2\.0-service\.intel u:object_r:hal_thermal_intel_exec:s0
+/vendor/bin/hw/android\.hardware\.thermal@aidl-service\.intel u:object_r:hal_thermal_intel_exec:s0
-- 
2.17.1

